packages/buidler-core/src/internal/buidler-evm/stack-traces/solidityTracer.ts

```diff
@@ -1471,7 +1455,7 @@ export class SolidityTracer {
   ) {
     // TODO: Change this when we add support for multiple compilations. This
```

Comment:
We should remove this TODO

Reply:
This is the change it actually describes

Reply:
Removed.